### PR TITLE
magpie release 4.6.4

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**

### CODE REVIEW

**Review by flohump — APPROVED** [review]
This PR includes a lot of changed files, mostly because of the tab-to-space modification.
I confirm that pfuehrlich-pik/develop is consistent with magpiemodel/develop

**Comment by pfuehrlich-pik** [comment]
Thanks! You can also ignore whitespace changes on github:
image

File: config/default.cfg [hunk 1]
```diff
@@ -125,7 +125,7 @@ cfg$gms$c_timesteps <- "coup2100"
 cfg$gms$c_past <- "till_2010"
 
 # use of gdx files
-cfg$gms$s_use_gdx <- 2  # def = 2
```
Comment (caviddhen) [inline config/default.cfg:128]: this change is intended right?

Comment (pfuehrlich-pik) [reply config/default.cfg:128]: I don't know who introduced this change, but it's already on magpiemodel/develop. The only changes I made for the release are [here](https://github.com/magpiemodel/magpie/compare/develop...pfuehrlich-pik:magpiemodel:develop)

Comment (pfuehrlich-pik) [reply config/default.cfg:128]: There's an entry for that in the changelog, so I'm pretty sure it is intended

**Review by tscheypidi — APPROVED** [review]

**Pull request merged**